[2016-11-02 01:25:34] application.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /" (uncaught exception) at /Users/solohin/Sites/toptal.local/backend/vendor/symfony/http-kernel/EventListener/RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /Users/solohin/Sites/toptal.local/backend/vendor/symfony/http-kernel/EventListener/RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/\". at /Users/solohin/Sites/toptal.local/backend/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2016-11-02 01:25:34] application.ERROR: No route found for "GET /" [] []
[2016-11-02 01:25:34] application.ERROR: #0 [internal function]: Symfony\Component\HttpKernel\EventListener\RouterListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher)) #1 /Users/solohin/Sites/toptal.local/backend/vendor/symfony/event-dispatcher/EventDispatcher.php(174): call_user_func(Array, Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher)) #2 /Users/solohin/Sites/toptal.local/backend/vendor/symfony/event-dispatcher/EventDispatcher.php(43): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #3 /Users/solohin/Sites/toptal.local/backend/vendor/symfony/http-kernel/HttpKernel.php(129): Symfony\Component\EventDispatcher\EventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #4 /Users/solohin/Sites/toptal.local/backend/vendor/symfony/http-kernel/HttpKernel.php(68): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #5 /Users/solohin/Sites/toptal.local/backend/vendor/silex/silex/src/Silex/Application.php(496): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #6 /Users/solohin/Sites/toptal.local/backend/vendor/silex/silex/src/Silex/Application.php(477): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request)) #7 /Users/solohin/Sites/toptal.local/backend/web/index.php(12): Silex\Application->run() #8 {main} [] []
[2016-11-02 01:25:50] application.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /" (uncaught exception) at /Users/solohin/Sites/toptal.local/backend/vendor/symfony/http-kernel/EventListener/RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /Users/solohin/Sites/toptal.local/backend/vendor/symfony/http-kernel/EventListener/RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/\". at /Users/solohin/Sites/toptal.local/backend/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2016-11-02 01:25:50] application.ERROR: No route found for "GET /" [] []
[2016-11-02 01:25:50] application.ERROR: #0 [internal function]: Symfony\Component\HttpKernel\EventListener\RouterListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher)) #1 /Users/solohin/Sites/toptal.local/backend/vendor/symfony/event-dispatcher/EventDispatcher.php(174): call_user_func(Array, Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher)) #2 /Users/solohin/Sites/toptal.local/backend/vendor/symfony/event-dispatcher/EventDispatcher.php(43): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #3 /Users/solohin/Sites/toptal.local/backend/vendor/symfony/http-kernel/HttpKernel.php(129): Symfony\Component\EventDispatcher\EventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #4 /Users/solohin/Sites/toptal.local/backend/vendor/symfony/http-kernel/HttpKernel.php(68): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #5 /Users/solohin/Sites/toptal.local/backend/vendor/silex/silex/src/Silex/Application.php(496): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #6 /Users/solohin/Sites/toptal.local/backend/vendor/silex/silex/src/Silex/Application.php(477): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request)) #7 /Users/solohin/Sites/toptal.local/backend/web/index.php(12): Silex\Application->run() #8 {main} [] []
[2016-11-02 01:25:53] application.ERROR: Symfony\Component\HttpKernel\Exception\NotFoundHttpException: No route found for "GET /" (uncaught exception) at /Users/solohin/Sites/toptal.local/backend/vendor/symfony/http-kernel/EventListener/RouterListener.php line 125 {"exception":"[object] (Symfony\\Component\\HttpKernel\\Exception\\NotFoundHttpException(code: 0): No route found for \"GET /\" at /Users/solohin/Sites/toptal.local/backend/vendor/symfony/http-kernel/EventListener/RouterListener.php:125, Symfony\\Component\\Routing\\Exception\\ResourceNotFoundException(code: 0): No routes found for \"/\". at /Users/solohin/Sites/toptal.local/backend/vendor/symfony/routing/Matcher/UrlMatcher.php:98)"} []
[2016-11-02 01:25:53] application.ERROR: No route found for "GET /" [] []
[2016-11-02 01:25:53] application.ERROR: #0 [internal function]: Symfony\Component\HttpKernel\EventListener\RouterListener->onKernelRequest(Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher)) #1 /Users/solohin/Sites/toptal.local/backend/vendor/symfony/event-dispatcher/EventDispatcher.php(174): call_user_func(Array, Object(Symfony\Component\HttpKernel\Event\GetResponseEvent), 'kernel.request', Object(Symfony\Component\EventDispatcher\EventDispatcher)) #2 /Users/solohin/Sites/toptal.local/backend/vendor/symfony/event-dispatcher/EventDispatcher.php(43): Symfony\Component\EventDispatcher\EventDispatcher->doDispatch(Array, 'kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #3 /Users/solohin/Sites/toptal.local/backend/vendor/symfony/http-kernel/HttpKernel.php(129): Symfony\Component\EventDispatcher\EventDispatcher->dispatch('kernel.request', Object(Symfony\Component\HttpKernel\Event\GetResponseEvent)) #4 /Users/solohin/Sites/toptal.local/backend/vendor/symfony/http-kernel/HttpKernel.php(68): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #5 /Users/solohin/Sites/toptal.local/backend/vendor/silex/silex/src/Silex/Application.php(496): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #6 /Users/solohin/Sites/toptal.local/backend/vendor/silex/silex/src/Silex/Application.php(477): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request)) #7 /Users/solohin/Sites/toptal.local/backend/web/index.php(12): Silex\Application->run() #8 {main} [] []
[2016-11-02 01:26:49] application.CRITICAL: Doctrine\DBAL\Exception\NotNullConstraintViolationException: An exception occurred while executing 'INSERT INTO notes (note) VALUES (?)' with params [null]:  SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: notes.note (uncaught exception) at /Users/solohin/Sites/toptal.local/backend/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractSQLiteDriver.php line 54 {"exception":"[object] (Doctrine\\DBAL\\Exception\\NotNullConstraintViolationException(code: 0): An exception occurred while executing 'INSERT INTO notes (note) VALUES (?)' with params [null]:\n\nSQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: notes.note at /Users/solohin/Sites/toptal.local/backend/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/AbstractSQLiteDriver.php:54, Doctrine\\DBAL\\Driver\\PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: notes.note at /Users/solohin/Sites/toptal.local/backend/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:93, PDOException(code: 23000): SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: notes.note at /Users/solohin/Sites/toptal.local/backend/vendor/doctrine/dbal/lib/Doctrine/DBAL/Driver/PDOStatement.php:91)"} []
[2016-11-02 01:26:49] application.ERROR: An exception occurred while executing 'INSERT INTO notes (note) VALUES (?)' with params [null]:  SQLSTATE[23000]: Integrity constraint violation: 19 NOT NULL constraint failed: notes.note [] []
[2016-11-02 01:26:49] application.ERROR: #0 /Users/solohin/Sites/toptal.local/backend/vendor/doctrine/dbal/lib/Doctrine/DBAL/DBALException.php(128): Doctrine\DBAL\Driver\AbstractSQLiteDriver->convertException('An exception oc...', Object(Doctrine\DBAL\Driver\PDOException)) #1 /Users/solohin/Sites/toptal.local/backend/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php(996): Doctrine\DBAL\DBALException::driverExceptionDuringQuery(Object(Doctrine\DBAL\Driver\PDOSqlite\Driver), Object(Doctrine\DBAL\Driver\PDOException), 'INSERT INTO not...', Array) #2 /Users/solohin/Sites/toptal.local/backend/vendor/doctrine/dbal/lib/Doctrine/DBAL/Connection.php(696): Doctrine\DBAL\Connection->executeUpdate('INSERT INTO not...', Array, Array) #3 /Users/solohin/Sites/toptal.local/backend/src/App/Services/NotesService.php(20): Doctrine\DBAL\Connection->insert('notes', Array) #4 /Users/solohin/Sites/toptal.local/backend/src/App/Controllers/NotesController.php(33): App\Services\NotesService->save(Array) #5 [internal function]: App\Controllers\NotesController->save(Object(Symfony\Component\HttpFoundation\Request)) #6 /Users/solohin/Sites/toptal.local/backend/vendor/symfony/http-kernel/HttpKernel.php(153): call_user_func_array(Array, Array) #7 /Users/solohin/Sites/toptal.local/backend/vendor/symfony/http-kernel/HttpKernel.php(68): Symfony\Component\HttpKernel\HttpKernel->handleRaw(Object(Symfony\Component\HttpFoundation\Request), 1) #8 /Users/solohin/Sites/toptal.local/backend/vendor/silex/silex/src/Silex/Application.php(496): Symfony\Component\HttpKernel\HttpKernel->handle(Object(Symfony\Component\HttpFoundation\Request), 1, true) #9 /Users/solohin/Sites/toptal.local/backend/vendor/silex/silex/src/Silex/Application.php(477): Silex\Application->handle(Object(Symfony\Component\HttpFoundation\Request)) #10 /Users/solohin/Sites/toptal.local/backend/web/index.php(12): Silex\Application->run() #11 {main} [] []
